### PR TITLE
USSD Registration - Update message receivers

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/ussd_public.js [hunk 1]
```diff
@@ -293,17 +271,39 @@ go.app = function() {
...
+                        if (contact.details.household_msgs_only) {
+                            self.im.user.set_answer('receiver_household_only', true);
+                            return self.states.create('state_main_menu_household');
+                        } else {
+                            return self.states.create('state_main_menu');
+                        }
+                    });
+            }
+
+            // return go.utils_project
```
Comment (imsickofmaps) [inline src/ussd_public.js]: Remove?

File: src/ussd_registration.js [hunk 2]
```diff
@@ -198,25 +201,39 @@ go.app = function() {
...
+                next: 'state_msisdn_household'
             });
         });
 
         // FreeText st-3A
-        self.add('state_msisdn_mother', function(name) {
+        self.add('state_msisdn_household', function(name) {
             return new FreeText(name, {
-                question: $(questions[name]),
+                question: $(questions[name]).context({
```
Comment (imsickofmaps) [inline src/ussd_registration.js:211]: Please just comment as to where this value is coming from to explain the complexity

**Comment by imsickofmaps** [comment]
Just the minor clarification then :+1:

**Pull request merged**